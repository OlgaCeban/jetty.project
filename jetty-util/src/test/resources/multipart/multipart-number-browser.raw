--RjYiyHVV9Phv7tYylzT1f94fvTC-s7oNKwB9_Y
Content-Disposition: form-data; name="pi"

3.14159265358979323846264338327950288419716939937510
--RjYiyHVV9Phv7tYylzT1f94fvTC-s7oNKwB9_Y--
