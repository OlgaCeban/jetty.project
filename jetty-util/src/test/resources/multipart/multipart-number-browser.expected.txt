Content-Type|multipart/form-data; boundary="RjYiyHVV9Phv7tYylzT1f94fvTC-s7oNKwB9_Y"
Parts-Count|1
Part-ContainsContents|pi|3.14159265358979323846264338327950288419716939937510
